system/hardware/tici/hardware.py

```diff
@@ -83,6 +83,17 @@ def affine_irq(val, action):
   for i in irqs:
     sudo_write(str(val), f"/proc/irq/{i}/smp_affinity_list")
 
+@lru_cache
```

Comment:
what's the reason on this one? seems worse

Reply:
# lru_cache and cache can cause memory leaks when used in classes

Reply:
https://beta.ruff.rs/docs/rules/cached-instance-method/

Reply:
oh, this happens to be fine here, but seems like a good rule to have